ETH-Merge-Bots/Milestones-in-TTD/README.md

```diff
@@ -0,0 +1,84 @@
+# Milestones in Terminal Total Difficulty
+
+## Description
+
+This bot monitors `totalDifficulty` of Ethereum blocks and calculates the estimated merge date by checking how close `totalDifficulty` is to `Terminal Total Difficulty`.
+In order to find the estimated merge date, the difference between `Terminal Total Difficulty` and `totalDifficulty` is divided by the `average block difficulty` which is calculated by the last 100 blocks. This returns the estimated number of blocks left to be mined. This value is multiplied by the `average block time` which is calculated by the most recent week's number of blocks and their average block time. To make this value dynamic, new block times are used to change `average block time` accordingly.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- ETH-1-1
+
+  - Fired when there are `20`, `15`, `10`, and `5-1` days left until the merge
+    - One alert will be emitted for each day below 5
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - metadata:
+    - `estimatedMergeDate`: Estimated merge date in UTC format.
```

Comment:
```suggestion
    - `estimatedMergeDate`: Estimated merge date in GMT.
```
While they are mostly the same, the finding actually displays the date and time in the GMT timezone.